.travis.yml

```diff
@@ -5,12 +5,16 @@ matrix:
   fast_finish: true
   include:
     - os: linux
+      env: OSNAME=linux_or
```

Comment:
better use `openresty` instead of `or`,  abbreviation is liable to cause misunderstanding.

Reply:
good, fixed.